### PR TITLE
disable rules added biome changes

### PR DESCRIPTION
**Pull request by DZotoff**

### CODE REVIEW

File: src/components/charts/timebank-piechart.tsx [hunk 1]
```diff
@@ -42,15 +42,14 @@ const TimebankPieChart = ({ personDailyEntry }: Props) => {
...
+              return null;
             }
+            return renderCustomizedLabel(entry);
           }}
           labelLine={false}
-        >     
+        >
           {dailyEntryToChart(personDailyEntry).map((_entry, index) => (
-            <Cell key={`cell-${index}`} fill={COLORS[index]} />
+            <Cell key={`cell-${_entry.name}`} fill={COLORS[index]} />
```
Comment (Jdallos) [inline src/components/charts/timebank-piechart.tsx:52]: Are you sure entry.name is unique. Do we have entry.id?
Also _ is used to indicate an unused argument, as it is being used here now we can remove the _

File: src/components/screens/timebank-view-all-screen.tsx [hunk 2]
```diff
@@ -107,7 +117,7 @@ const TimebankViewAllScreen = () => {
             label={(props) => getHoursAndMinutes(props.value)}
           >
             {workTimeData.map((_entry, index) => (
-              <Cell key={`cell-${index}`} fill={COLORS[index % COLORS.length]} />
+              <Cell key={`cell-${_entry.name}`} fill={COLORS[index % COLORS.length]} />
```
Comment (Jdallos) [inline src/components/screens/timebank-view-all-screen.tsx:120]: Again no need for _ if we are using entry. Do we not have entry.id?

**Review by Jdallos — APPROVED** [review]
Approved with a couple of minor comments

File: src/components/charts/timebank-piechart.tsx (hunk 1 above)
Comment (DZotoff) [reply src/components/charts/timebank-piechart.tsx:52]: There is no entry.id property

**Pull request merged**